runelite-mixins/src/main/java/net/runelite/mixins/RSDecorativeObjectMixin.java

```diff
@@ -110,7 +110,29 @@ private RSModel getModel2()
 	@Override
 	public Area getClickbox()
 	{
-		return Perspective.getClickbox(client, getModel(), getOrientation(), getLocalLocation());
+		Area clickbox = new Area();
+
+		LocalPoint lp = getLocalLocation();
+		Area clickboxA = Perspective.getClickbox(client, getModel(), 0,
+			new LocalPoint(lp.getX() + getXOffset(), lp.getY() + getYOffset()));
```

Comment:
This offset is actually only sometimes applied. I don't quite understand it but it has something to do with the camera angle. I think it is so if you are facing the wall behind it it won't draw through the wall.

Reply:
I don't see where the offset is conditionally applied. I only see where it conditionally renders the primary or secondary renderable.